nemo/collections/nlp/models/nlp_model.py

```diff
@@ -166,7 +166,7 @@ def register_bert_model(self):
                         encoder_config_src = os.path.join(NEMO_NLP_TMP, encoder_config_path + '.json')
                         with open(encoder_config_src, 'w', encoding='utf-8') as f:
                             f.write(json.dumps(output_config, indent=2, sort_keys=True) + '\n')
-                        self.register_artifact('language_model.config_file', encoder_config_src)  # for .nemo
+                        # self.register_artifact('language_model.config_file', encoder_config_src)  # for .nemo
```

Comment:
Can you check RIVA compliance for a Megatron model, I think this maybe required for RIVA compliance

Reply:
@borisfom , could you help to verify the impact of this change on RIVA?